docs/concepts/portal/installation-portal.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Ondat Portal Installation Guide"
+linkTitle: "Ondat Portal Installation Guide"
+---
+
+
+# Setting Up a Local Cluster 
```

Comment:
This should be marked as an optional step.

Reply:
I think all the _Local Cluster_ steps should be removed. They're likely to be more confusing than helpful. I have a copy of the `cloud_init.yml` file but it's not up to a standard where we can release it publicly.

These headings and their content should be removed: 
- `# Setting Up a Local Cluster`
- `## Prerequisites`
- `## Procedure`

Maybe there could be a new _Prerequisites_ section, that says something like:

_You will need access to a Kubernetes cluster that meets the [prerequisites for installing Ondat](https://docs.ondat.io/docs/prerequisites/)._